README.md

```diff
@@ -118,28 +118,27 @@ Contributions are highly welcome.
 
 ### Queries
 
-    Query files live in the `src/queries` directory. They are static resources; that are loaded into `run-query` and
-    then sent to BigQuery for actual execution. It is up to the developer to ensure their query is correct; this 
-    can be done by using the BigQuery console.
-
-    Once a query file is complete and correct, you may add it as a static resource; so that it won't be excluded during OpenWhisk deployment.
-    In the root of the repository; find the package.json and add your query file (file with `.sql` extension) under `static`: 
-
-    ```json
-    "wsk": {
-    "name": "helix-services/run-query@${version}",
-        "static": [
-            "src/queries/next-resource.sql"
-        ]
-    },
-    ```
-
-    Now, query file can be executed as an action; triggered by a request as such: 
-
-    ```bash
-    curl -X POST -H "Content-Type: application/json" https://adobeioruntime.net/api/v1/w/helix-services/run-query@v1/next-resource -d 
-    '{"service":"secretService", "token":"secretToken", "queryParam1":"value"}'
-    ```
+Query files live in the `src/queries` directory. They are static resources; that are loaded into `run-query` and
+then sent to BigQuery for actual execution. It is up to the developer to ensure their query is correct; this 
+can be done by using the BigQuery console.
+
+Once a query file is complete and correct, you may add it as a static resource; so that it won't be excluded during OpenWhisk deployment.
+In the root of the repository; find the package.json and add your query file (file with `.sql` extension) under `static`: 
+
+```json
+"wsk": {
+"name": "helix-services/run-query@${version}",
+"static": [
+     "src/queries/next-resource.sql"
```

Comment:
Maybe we should use wildcards here. @tripodsan are wildcards supported by wsk-builder?